lua/dylib.lua

```diff
@@ -0,0 +1,20 @@
+module(..., package.seeall)
+
+--
+-- C functions exported through FFI
+--
+
+local ffi = require("ffi")
+
+-- Structs
+ffi.cdef[[
+
+extern const char *CLIENT_PROC_ERROR;
+
+enum dyn_cmd_op {
```

Comment:
I think if we move the code that would be inside ``dy_cmd_proc()`` to the end of the client's request chunk, then we don't need the ``enum`` here, since the client request chunk would contain the code to call corresponding block's functions directly.